common/src/sql/dbinit.sql

```diff
@@ -178,14 +175,10 @@ CREATE TABLE omicron.public.switch (
 );
 
 /* Add an index which lets us look up switches on a rack */
-CREATE INDEX ON omicron.public.switch (
-    rack_id
-) WHERE time_deleted IS NULL;
-
-CREATE INDEX ON omicron.public.switch (
+CREATE UNIQUE INDEX ON omicron.public.switch (
```

Comment:
Another of the cases mentioned in the PR description.  This one's arguably unnecessary since we don't expect to have many switches in a rack.